syslog-ng-config.h.in

```diff
@@ -58,4 +58,5 @@
 #cmakedefine01 SYSLOG_NG_HAVE_DECL_DH_SET0_PQG
 #cmakedefine01 SYSLOG_NG_HAVE_DECL_BN_GET_RFC3526_PRIME_2048
 #cmakedefine01 SYSLOG_NG_HAVE_INOTIFY
+#cmakedefine01 SYSLOG_NG_HAVE_GETRANDOM
 #cmakedefine01 SYSLOG_NG_USE_CONST_IVYKIS_MOCK
```

Comment:
isn't this a generated file? if it is, it shouldn't be committed into git.

Reply:
Autotools can generate it, but cmake does not. In case of cmake it needs a template (this  file) to substitute the values, and create the `syslog-ng.conf.h`.